src/TesApi.Web/Controllers/TaskServiceApi.cs

```diff
@@ -124,7 +126,26 @@ public virtual async Task<IActionResult> CreateTaskAsync([FromBody]TesTask tesTa
                 ?.Split('/', StringSplitOptions.RemoveEmptyEntries)
                 ?.Skip(2)
                 ?.FirstOrDefault();
-            
+
+            // For CWL workflows, if disk size is not specified in TES object (always), try to retrieve it from the corresponding workflow stored by Cromwell in /cromwell-tmp directory
+            // Also allow for TES-style "memory" and "cpu" hints in CWL.
+            if (tesTask.Name != null 
+                && tesTask.Name.EndsWith(".cwl") 
```

Comment:
&& tesTask.Name.EndsWith(".cwl", StringComparison.OrdinalIgnoreCase)

Reply:
Value is created by Cromwell, it is always lowercase.